### PR TITLE
[*] BO: Improved adding to Quick Access

### PR DESCRIPTION
**Pull request by kpodemski**
Previously adding page to Quick Access was really annoying because every time I needed to change name of the shortcut... now we can set name right before running a script.

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Sponsor company | impSolutions

### CODE REVIEW

File: admin-dev/themes/default/template/header.tpl [hunk 1]
```diff
@@ -230,19 +230,23 @@
...
 				</ul>
-				{$quick_access_current_link_name = " - "|explode:$quick_access_current_link_name}
 				<script>
 					$(function() {
 						$('.ajax-quick-link').on('click', function(e){
 							e.preventDefault();
+
+							var method = $(this).data('method');
+							if(method == 'add')
+								var name = prompt("{l s='Please provide name for this shortcut' js=1}");
```
Comment (jeromenadaud) [inline admin-dev/themes/default/template/header.tpl]: Hi @kpodemski,

maybe you can add ```$quick_access_current_link_name``` as default value and check if name is empty do not fire ajax.

Best regards.

Comment (kpodemski) [inline admin-dev/themes/default/template/header.tpl]: Hi @jeromenadaud 

I changed the code with your suggestion

**Comment by jeromenadaud** [comment]
:+1:

**Comment by jeromenadaud** [comment]
Thank you for your contribution @kpodemski 
Best regards.

**Pull request merged**